docs/codeql_core.md

```
@@ -0,0 +1,51 @@
+      contents: read
+      security-events: write
+    uses: masutaka/actions/.github/workflows/codeql_core.yml@main
+    with:
+      language: ${{ matrix.language }}
+```
+
+## 関連
+
+- [Finding security vulnerabilities and errors in your code with code scanning - GitHub Docs](https://docs.github.com/code-security/code-scanning)
+- [CodeQL overview — CodeQL](https://codeql.github.com/docs/codeql-overview/)
+- https://github.com/github/codeql-action
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)